java/src/main/native/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 # =============================================================================
-cmake_minimum_required(VERSION 3.26.4 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.30.4 FATAL_ERROR)
```

Comment:
Oh sorry, you need to update `CMAKE_VERSION` in the docker file (`Dockerfile.rocky`).

Reply:
Done